README.md

```diff
@@ -56,7 +56,11 @@ The table below lists the recommender algorithms currently available in the repo
 | Factorization Machine (FM) / Field-Aware FM (FFM) | [Python CPU](notebooks/02_model/fm_deep_dive.ipynb) | Content-Based Filtering | Algorithm that predict labels with user/item features |
 | FastAI Embedding Dot Bias (FAST) | [Python CPU / Python GPU](notebooks/00_quick_start/fastai_movielens.ipynb) | Collaborative Filtering | General purpose algorithm with embeddings and biases for users and items |
 | LightGBM/Gradient Boosting Tree<sup>*</sup> | [Python CPU](notebooks/00_quick_start/lightgbm_tinycriteo.ipynb) / [PySpark](notebooks/02_model/mmlspark_lightgbm_criteo.ipynb) | Content-Based Filtering | Gradient Boosting Tree algorithm for fast training and low memory usage in content-based problems |
+| Neural News Recommendation with Long- and Short-term User Representations (LSTUR) | [Python CPU / Python GPU](notebooks/00_quick_start/lstur_synthetic.ipynb) | Content-Based Filtering | Deep learning news recommendation algorithm considering users' long- and short-term preference |
+| Neural News Recommendation with Attentive Multi-View Learning (NAML) | [Python CPU / Python GPU](notebooks/00_quick_start/naml_synthetic.ipynb) | Content-Based Filtering | Deep learning news recommendation algorithm with multi-view learning |
 | Neural Collaborative Filtering (NCF) | [Python CPU / Python GPU](notebooks/00_quick_start/ncf_movielens.ipynb) | Collaborative Filtering | Deep learning algorithm with enhanced performance for implicit feedback |
+| Neural News Recommendation with Personalized Attention (NPA) | [Python CPU / Python GPU](notebooks/00_quick_start/npa_synthetic.ipynb) | Content-Based Filtering | Deep learning news recommendation algorithm ultilizing personalized attention |
+| Neural News Recommendation with Multi-Head Self-Attention (NRMS) | [Python CPU / Python GPU](notebooks/00_quick_start/nrms_synthetic.ipynbb) | Content-Based Filtering | Deep learning news recommendation algorithm ultilizing multi-head self-attention |
```

Comment:
The rest of the algorithms in this table are not domain specific. Is it important to have the news focus here? It may lead to people overlooking it as an option for other domains.